RecoTracker/DeDx/plugins/DeDxEstimatorProducer.cc

```diff
@@ -34,8 +34,8 @@ void DeDxEstimatorProducer::fillDescriptions(edm::ConfigurationDescriptions& des
   desc.add<edm::InputTag>("tracks",edm::InputTag("generalTracks"));
   desc.add<bool>("UsePixel",false); 
   desc.add<bool>("UseStrip",true); 
-  desc.add<double>("MeVperADCPixel",3.61e-06*265);
-  desc.add<double>("MeVperADCStrip",3.61e-06);
+  desc.add<double>("MeVperADCPixel",3.61e-06);
+  desc.add<double>("MeVperADCStrip",3.61e-06*265);
```

Comment:
this change does not belong in this PR.
Please rebase appropriately to get rid of this change.